microservice/challenge-1/app42/frontend/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from 'react';
```

Comment:
You seem to have some knowledge on TypeScript and React, do you think it's possible to have the frontend application use both? Also, is there any difference between `js` and `jsx` file extension for React?

Reply:
Yes, these two technologies could be used together, but I prefer use JSX for react.
And there will be huge different in using JSX. 

Shamefully, for the frontend, I just wanted to deliver the code.

Reply:
But in this case, your files are all using `.js` extension, instead of the `.jsx`, is there any reason, since I don't think it introduce more hiccups or demand more time 🤔